felles/gradle.properties

```diff
@@ -1,6 +1,6 @@
 # Dependency versions
-lettuceVersion=6.3.2.RELEASE
+lettuceVersion=6.5.0.RELEASE
 prometheusVersion=0.16.0
-rapidsAndRiversTestVersion=2024.09.23-08.33-8839f9a1
+rapidsAndRiversTestVersion=2024.11.19-09.27-9c591574
 rapidsAndRiversVersion=2024082715011724763661.50d7efb40f0f
```

Comment:
Oppdaterer ikke denne fordi det ofte fører med seg krøll. Bør nok tas for seg selv.

Reply:
test brakk også 😐

Reply:
Fjernet oppdateringen av testversjonen av R&R også, fordi det så ut som at avhengigheter de delte hadde ulike versjoner.